### PR TITLE
[python] improving the syntax of the fstring in the file : tests/python_package_test/test_dask.py

### PR DESCRIPTION
**Pull request by sayantan1410**
I have updated the old python syntax with the fstrings in the file : tests/python_package_test/test_dask.py
The pull request is contributing to #4136

### CODE REVIEW

**Review by jameslamb — CHANGES_REQUESTED** [review]
Thanks very much! Can you please also convert the places where `+` concatenation is being used in this file?

https://github.com/microsoft/LightGBM/blob/bab58d0e90b2ee93d7e0656957e73e92d46554bc/tests/python_package_test/test_dask.py#L98

https://github.com/microsoft/LightGBM/blob/bab58d0e90b2ee93d7e0656957e73e92d46554bc/tests/python_package_test/test_dask.py#L175

https://github.com/microsoft/LightGBM/blob/bab58d0e90b2ee93d7e0656957e73e92d46554bc/tests/python_package_test/test_dask.py#L1108

https://github.com/microsoft/LightGBM/blob/bab58d0e90b2ee93d7e0656957e73e92d46554bc/tests/python_package_test/test_dask.py#L1154

https://github.com/microsoft/LightGBM/blob/bab58d0e90b2ee93d7e0656957e73e92d46554bc/tests/python_package_test/test_dask.py#L1174

**Comment by sayantan1410** [comment]
yeah sure I will do it.

**Comment by sayantan1410** [comment]
Hey, please check if I have done that correctly. 
@jameslamb

File: tests/python_package_test/test_dask.py [hunk 1]
```diff
@@ -1105,7 +1105,7 @@ def test_network_params_not_required_but_respected_if_given(task, listen_port, c
             n_estimators=5,
             num_leaves=5,
             machines=",".join([
-                "127.0.0.1:" + str(port)
+                f"127.0.0.1:{str(port)}"
```
Comment (jameslamb) [inline tests/python_package_test/test_dask.py:1108]:
```suggestion
                f"127.0.0.1:{port}"
```

File: tests/python_package_test/test_dask.py [hunk 2]
```diff
@@ -1171,7 +1171,7 @@ def test_machines_should_be_used_if_provided(task, cluster):
         one_open_port = lgb.dask._find_random_open_port()
         dask_model.set_params(
             machines=",".join([
-                "127.0.0.1:" + str(one_open_port)
+                f"127.0.0.1:{str(one_open_port)}"
```
Comment (jameslamb) [inline tests/python_package_test/test_dask.py:1174]:
```suggestion
                f"127.0.0.1:{one_open_port}"
```

**Review by jameslamb — CHANGES_REQUESTED** [review]
thanks! Please see the newest round of comments. `str()` is not necessary inside f-strings.

**Review by jameslamb — APPROVED** [review]
looks good, thanks!

In the future, when you get comments with code suggestions like this:

image

I think you might find it is easier to just click "add to batch" on all of them, then "commit suggestions". Doing that automatically resolves /hides the comment threads, and reduces the risk of forgetting to make one of the changes. It is also a little easier for maintainers to see that their suggestions have been accepted.

**Comment by sayantan1410** [comment]
Thank you so much, I will keep that in mind and from next time onward will do that. 
@jameslamb

**Pull request merged**